test/classes-additional.coffee

```diff
@@ -0,0 +1,301 @@
+# Classes
+# -------
+
+# * Class Instantiation
+# * Inheritance and Super
+# * Default parameters
+# * Version Compatability
+test "multiple super calls", ->
+  # class A 
+  #   constructor: (@drink) ->
+  #   make: -> "Making a #{@drink}"
+
+  # class MultiSuper extends A
+  #   constructor: (drink) ->
+  #     super(drink)
+  #     super(drink)
+  #     @newDrink = drink
+  # eq (new MultiSuper('Late')).make(), 'Making a Late'
+
+test "@ params", ->
+  class A 
+    constructor: (@drink, @shots, @flavor) ->
+    make: -> "Making a #{@flavor} #{@drink} with #{@shots} shot(s)"
+
+  a = new A('Machiato', 2, 'chocolate')
+  eq a.make(),  "Making a chocolate Machiato with 2 shot(s)"
+
+  class B extends A
+  b = new B('Machiato', 2, 'chocolate')
+  eq b.make(),  "Making a chocolate Machiato with 2 shot(s)"
+
+test "@ params with defaults", ->
+  class A 
+    # Multiple @ params with defaults
+    constructor: (@drink = 'Americano', @shots = '1', @flavor = 'caramel') ->
+    make: -> "Making a #{@flavor} #{@drink} with #{@shots} shot(s)"
+
+  a = new A()
+  eq a.make(),  "Making a caramel Americano with 1 shot(s)"
+
+test "@ params with class params", ->
```

Comment:
Not sure about the name of this test, seems to be a variant of `"@ params with defaults"`.